assets/js/blocks/active-filters/block.js

```diff
@@ -117,19 +125,34 @@ const ActiveFiltersBlock = ( {
 	}
 
 	const TagName = `h${ blockAttributes.headingLevel }`;
-	const listClasses = classnames( 'wc-block-active-filters__list', {
-		'wc-block-active-filters__list--chips':
-			blockAttributes.displayStyle === 'chips',
-	} );
+	const listClasses = classnames(
+		'wc-block-active-filters__list',
+		textColorClass,
+		{
+			'wc-block-active-filters__list--chips':
+				blockAttributes.displayStyle === 'chips',
+		}
+	);
 
 	return (
 		<>
 			{ ! isEditor && blockAttributes.heading && (
-				<TagName className="wc-block-active-filters__title">
+				<TagName
+					className={ classnames(
+						'wc-block-active-filters__title',
+						textColorClass
+					) }
+					style={ textColorStyle }
```

Comment:
Is it necessary to pass the `style` and the `textColorClass`? I would expect the heading element to simply inherit the color of the parent without us needing to do anything special.